.github/workflows/release.yml

```diff
@@ -6,31 +6,13 @@ name: OpenJVerein official release
 on: workflow_dispatch
 
 jobs:
-  call-reusable-workflow:
-    uses: ./.github/workflows/reusable-build.yml
-
   release:
-    needs: call-reusable-workflow
     runs-on: ubuntu-latest
     steps:
-    - name: Restore cached tags and jars
-      id: cache-tags-jars
-      uses: actions/cache@v4
-      with:
-        path: |
-          ./cached-tags
-          jameica
-          hibiscus
-        key: ${{ runner.os }}-${{ github.repository_id }}
-
-    - name: Checkout openjverein
-      uses: actions/checkout@v4
```

Comment:
Da hast du Zuviel gelöscht. Auschecken muss man vorher schon im Workflow, sonst findet er die Action nicht. Dafür braucht man es nicht mehr in der Action auszuchecken.